crates/enarx-config/src/lib.rs

```diff
@@ -1,5 +1,12 @@
 // SPDX-License-Identifier: Apache-2.0
-//! The Enarx Configuration file format
+
+//! Configuration for a WASI application in an Enarx Keep
```

Comment:
What was wrong with the original comment?
Do we need to call workloads "WASI applications"? Why not just Enarx workload or Enarx Keep configuration?

Reply:
Hmm, because it's only relevant for exec-wasmtime